microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -3,89 +3,119 @@ description: Version Models used for Microsoft Edge WebView2
+> When developing [Evergreen WebView2 apps][Webview2ConceptsDistributionEvergreenDistributionMode], regularly test your app against the latest non-stable Microsoft Edge channels and the WebView2 Runtime.  Because the web platform is constantly evolving, regular testing is the best way to ensure your app performs as intended.
 
-### Determine WebView2 Runtime requirement  
 
-Depending on which SDK you use, consider the following items:  
+## Testing whether the installed Runtime supports a recent API
 
-*   **Win32 C/C++**.  When using `QueryInterface` to obtain a new interface, verify a return value of `E_NOINTERFACE`.  The value may indicate that the WebView2 Runtime is a previous version, and doesn't support that interface.  For an example of how it works, navigate to [Line 622 - AppWindow.cpp][GithubMicrosoftedgeWebview2samplesSampleappsWebview2apisampleAppwindowCppL622].  
-*   **.NET and WinUI**.  Check for a `No such interface supported` exception when using methods, properties, and events that were added to more recent SDKs.  The exception may occur when the WebView2 Runtime is a previous version, and does not support the APIs.  
+When you develop a WebView2 app using a recent version of the WebView2 SDK, when you use a recently added API, it's a good practice to test the client's installed WebView2 Runtime to check whether that API is present.  How your app programmatically tests for API support depends on the coding platform.
+
+*   **Win32 C/C++**.  When using `QueryInterface` to obtain a new interface, test for a return value of `E_NOINTERFACE`.  That return value likely indicates that the client's WebView2 Runtime is an older version that doesn't support that interface.  For an example of checking for the existence of specific WebView2 APIs in the Runtime, navigate to [Line 622 - AppWindow.cpp][GithubMicrosoftedgeWebview2samplesSampleappsWebview2apisampleAppwindowCppL622].<!-- outdated?  that cpp file contains neither "catch" nor "QueryInterface" nor "E_NOINTERFACE" -->
+
+*   **.NET and WinUI**.  Use `try/catch` and check for a `No such interface supported` exception when using methods, properties, and events that were added to more recent versions of the WebView2 SDK.  This exception likely indicates that the client's WebView2 Runtime is an older version that doesn't support that API.
     
-If an API is unavailable, consider removing the associated feature, or inform your users that an update to the WebView2 Runtime is required.  
+If an API is unavailable in the client's installed WebView2 Runtime, consider turning off the associated feature for the user, or inform the user that they must update the WebView2 Runtime to use the feature.
+
 
-<!--
+<!-- Is this section commented out because deprecated?  delete or uncomment the section?
 ## Versioning  
```

Comment:
Is this section commented out because deprecated? delete or uncomment the section?

Reply:
I rewrote but kept section commented-out.